etna/models/holt_winters.py

```diff
@@ -316,19 +332,24 @@ def forecast_components(self, df: pd.DataFrame) -> pd.DataFrame:
         model = self._model
         fit_result = self._result
 
-        if fit_result is None or model is None:
+        if fit_result is None or model is None or self._train_freq is None:
             raise ValueError("This model is not fitted!")
 
+        if df["timestamp"].max() <= self._last_train_timestamp:
```

Comment:
What if `df["timestamp"].min() <= self._last_train_timestamp`? Shouldn't all the timestamps be out of sample?

Reply:
Sure, it should be `min` to cover all cases